src/main/resources/org/jenkinsci/plugins/workflow/libs/ResourceStep/config.jelly

```diff
@@ -28,4 +28,7 @@ THE SOFTWARE.
     <f:entry field="resource" title="Resource path">
         <f:textbox/>
     </f:entry>
+    <f:entry field="encoding" title="${%Character encoding}">
```

Comment:
I completely forgot to add this before, but thankfully caught it when updating the documentation.

Reply:
Probably because it's something that *ought* to have been there before.